src/pages/UserSettings/UserSettings.tsx

```diff
@@ -8,16 +8,17 @@ import TextField from "../../components/TextField";
 import Toggle from "../../components/Toggle";
 import Column from "../../components/layout/Column";
 import Row from "../../components/layout/Row";
-import ConfirmationModal from "../../components/modals/ConfirmationModal";
+import ClosableModal from "../../components/modals/ClosableModal";
 import { Constants } from "../../constants";
 import { useApi } from "../../contexts/ApiProvider";
 import { useUser } from "../../contexts/UserProvider";
 import { SettingShort } from "../../data/models/Setting";
 import { getTimeZone } from "../../utils";
+import DeleteAccountModals from "./DeleteAccountModals";
 import RecoveryPhraseModal from "./RecoveryPhraseModal";
 
 export default function UserSettings() {
```

Comment:
super minor thing but see the weird horizontal scroll issue that I sent on discord